javascript/object/object.ts

```diff
@@ -0,0 +1,35 @@
+interface IMusic {
+  title: string;
+  artist: string;
+  genre: string;
+}
+
+const musicLiteral: IMusic = {
+  title: "title",
+  artist: "artist",
+  genre: "genre",
+};
+
+const musicNewObject = new Object() as IMusic;
+musicNewObject.title = "title";
+musicNewObject.artist = "artist";
+musicNewObject.genre = "genre";
+
```

Comment:
p5: 이렇게 바로 정의할 수도 있겠네용

```suggestion
const musicFromClass = new MusicClass("title", "artist", "genre");


```